samples/jwt-server/jwt-server.yaml

```diff
@@ -106,7 +106,7 @@ spec:
         app: jwt-server
     spec:
       containers:
-      - image: gcr.io/istio-testing/jwt-server:0.7
+      - image: gcr.io/istio-testing/jwt-server:0.8
```

Comment:
This should be inconsistent with https://github.com/istio/istio/blob/d0909a630f8642f4f6158e546810e6350564a7ba/samples/jwt-server/src/Makefile

Reply:
Now it is consistent as I have rebased the commit, master already had 0.8 inside the makefile